charts/backup-operator/templates/deployment.yaml

```diff
@@ -28,6 +28,7 @@ spec:
         command:
         - /manager
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        worker-image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
```

Comment:
This is not valid in K8s deployment resources. The Worker image has to be set as an argument to the command above like for the leader election in line 27. The argument is called `--worker-image` and there you have to pass the value.